cli/packages/util/keyringwrapper.go

```diff
@@ -20,16 +27,32 @@ func SetValueInKeyring(key, value string) error {
 		PrintErrorAndExit(1, err, "Unable to get current vault. Tip: run [infisical rest] then try again")
 	}
 
-	return keyring.Set(currentVaultBackend, MAIN_KEYRING_SERVICE, key, value)
+	err = keyring.Set(currentVaultBackend, MAIN_KEYRING_SERVICE, key, value)
+
+	if err == keyring.ErrUnsupportedPlatform || keyringNotConfigured(err) {
```

Comment:
Looks like the default to file mode is not being set. You can reproduce it by logging in inside of codespaces
<img width="1087" alt="CleanShot 2024-07-26 at 19 22 07@2x" src="https://github.com/user-attachments/assets/867806ff-2f00-4971-bca9-4631f7e208c9">